### ISSUE TITLE
This is not the promblem with npm..There is additional logging output above...

### ISSUE THREAD

**Issue by sonaskyla12** [eslint/eslint#10501]
please help me ...currently i m working on windows
with npm -v =5.3.0 and node -v =v10.4.0 and i m getting this error
'nodemon' is not recognized as an internal and external command,
operable program or batch file.
npm ERR! code ELIFECYCLE
npm ERR! erno 1
npm ERR! mernstack@1.0.0 server : 'nodemon server js'
npm ERR! Exit status 1
npm ERR! failed at the mernstack@1.0.0 server script
npm ERR! this probably not the problem with npm .There is additional logging output above
npm ERR! A complete log of this run can be found in :
C:\User\shubhendu\APPdata\roaming\npm-cache_logs\2018-06-22T03_20_40_649Z-debug.log

**Comment by sonaskyla12** [comment]
please help me

**Comment by platinumazure** [comment]
Hi @sonaskyla12, thanks for the issue.

Please provide the information requested above (and please format it neatly so we can read it).

I understand you are struggling with a project, but please note that all of our team members help out as volunteers, in their spare time. In fact, I'm writing this post while lying in my bed, about to go to sleep. :smile: So please be patient with us (and help us out by providing the information we need), and we'll help you as soon as we can. Thanks!

**Comment by sonaskyla12** [comment]
thank you for your reply....
eslint -v is v4.19.1
npm -v =5.3.0 
 node -v =v10.4.0 
'nodemon' is not recognized as an internal and external command,
operable program or batch file.
npm ERR! code ELIFECYCLE
npm ERR! erno 1
npm ERR! mernstack@1.0.0 server : 'nodemon server js'
npm ERR! Exit status 1
npm ERR! failed at the mernstack@1.0.0 server script
npm ERR! this probably not the problem with npm .There is additional logging output above
npm ERR! A complete log of this run can be found in :
C:\User\shubhendu\APPdata\roaming\npm-cache_logs\2018-06-22T03_20_40_649Z-debug.log
I am getting this error whenever i m trying to do...
npm run server

**Comment by sonaskyla12** [comment]
currently i m working on (mern stack)
and now i m in middle of my process.. 
i was getting the above error when i did "npm run server"... 
Various code wriiten in this project from starting are:-
Step 1.
npm init
->after this my inital file was server.js
-> code under this was
const express = require('express');
const mongoose = require('mongoose');
const bodyparser = require('body-parser');
const users = require('./routes/api/users');
const profile = require('./routes/api/profile');
const posts = require('./routes/api/posts');
const app = express()
//body-parser middleware
app.use(bodyparser.urlencoded({extended : false}));
app.use(bodyparser.json());

//db config
const db = require('./config/keys').mongoURI;
//connect to mongodb
mongoose
.connect(db)
.then(() => console.log('MongoDB connected'))
.catch(err => console.log(err));


app.get('/',(req,res) =>res.send('Hello! world sonali '));
//use routes
app.use('/api/users', users);
app.use('/api/profile', profile);
app.use('/api/posts', posts);

const port = process.env.PORT || 5000;

app.listen(port,() => console.log('server running on port ${port}'));
->Various connection wriiten is used by my upcoming files..
step 2. connection of mongodb with mongoose in file "key.js"
module.exports = {

  mongoURI:'mongodb://sonali:skyla123@ds161580.mlab.com:61580/sonali'
};
Step 3:now made the folder under node_modules\routes\api\models\User.js.. and code under User.js are:-
const mongoose = require ('mongoose');
const Schema = mongoose.Schema;
//create schema
const UserSchema = new Schema({ 
 name:{
   type :  string,
   required : true
 },
email:{
   type :  string,
   required : true
 },
 password:{
   type :  string,
   required : true
 },
 avatar:{
   type :  string
 },
 date:{
   type :  Date,
   default: Date.now
 },

});
module.exports = User = mongoose.model('user',UserSchema);
Step 4 :-three folder for user identification ...(user.js,post.js,profile.js) for knowing the user accessibility,there post,and profile
(i) under the folder user.js code written are:-
const express = require ('express');
const router = express.Router();

//route Get api/user/test
router.get('/test', (req, res) => res.json({msg: 'user works'}));
module.exports = router;
(ii) under the folder profile.js
const express = require('express');
const router = express.Router();
//@route GET api/profile/test
//@desc Tests profile route
//@access public
router.get('/test', (req, res) => res.json({ msg: "profile Works" }));

module.exports = router;
(iii) the last folder was post.js and the code under that was:-
const express = require('express');
const router = express.Router();
//@route GET api/posts/test
//@desc Tests post route
//@access public
router.get('/test', (req, res) => res.json({ msg: "posts Works" }));

module.exports = router;

after typing npm run server i was showig the above problem

**Comment by sonaskyla12** [comment]
thank you sir for your earlier respose would wait for your next response to....

**Comment by sonaskyla12** [comment]
Earlier i was getting the output 
Mongodb connected.
and response was also send by postman..
but at the time i started my code in user.js folder for registration for user the above error started appearing...i erased the new line for registration and then:-
->after typing npm run server 
above error was still the same

**Comment by sonaskyla12** [comment]
earlier  i was getting the response after typing
on Vscode
npm run server
->and in the postman localhost:5000/api/user/test
but not now even i have made my code same as it was earlier

**Comment by mysticatea** [comment]
> 'nodemon' is not recognized as an internal and external command, operable program or batch file.

Could you explain the reason you though that the error is related to ESLint? I guess that the error is caused by that you used `nodemon` command but you didn't install it, based on the error message.

**Comment by sonaskyla12** [comment]
since i have already installed nodemon earlier and it was running till yesterday which i forgot to mentain about it..this is my code for package.json 
 {
  "name": "mernstack",
  "version": "1.0.0",
  "description": "social network for developers",
  "main": "server.js",
  "scripts": {
    "start": "node server.js",
    "server": "nodemon server.js"
  },
  "author": "sonali",
  "license": "MIT",
  "dependencies": {
    "gravatar": "^1.6.0"
  },
  "devDependencies": {
    "eslint": "^4.19.1"
  }
}.
.now i have made it global by npm install -g nodemon..
still i m showing the error
[nodemon] app crashed--waiting for file changes before starting

**Comment by mysticatea** [comment]
Would you explain the reason you thought that the error is related to ESLint?

**Comment by sonaskyla12** [comment]
i am getting this error after installing nodemon globally
internal/module/cjs/loader.js:596
throw err;
Err: cannot find the module'express'
at.Function.Module._resolveFilename(internal/module/cjs/loader.js:594:15)
at.Function.Module._load(internal/module/cjs/loader.js:594:25)
and at the end it is showing
[nodemon] app crashed--waiting for file changes before starting...
even though i  have install express earlier so we have to install there dependencies again

**Comment by mysticatea** [comment]
> cannot find the module'express'

Would you explain the reason you thought that the error is related to ESLint?
In other words, why do you think it's ESLint's error?

**Comment by sonaskyla12** [comment]
because the simpler error was written under ESlint  section  ....and since my server was running till yesterday...so i thought might be this cud be the reason

**Comment by sonaskyla12** [comment]
i was not sure about it ....

**Comment by sonaskyla12** [comment]
ESLint section mean various promblem related to it
now if your question are over could you please ans my question

**Comment by mysticatea** [comment]
Thanks. So this is not related to ESLint.

The first error said "you don't install `nodemon` package".
The second error said "you don't install `espress` package".

You don't seem to run ESLint.

**Comment by sonaskyla12** [comment]
ESLint section means various problem related because nodemon was running till yesterday so could you helpme...?

**Comment by sonaskyla12** [comment]
but i have installed all these earlier express,nodemon,mongo db.......
i m really sry i thought it was related to ESLint

**Comment by sonaskyla12** [comment]
does this mean i have to install all the dependencies again even though i have installed it earlier....???
ya  i know you are right i m not using ESLint ....

**Comment by sonaskyla12** [comment]
i am getting this error after installing nodemon globally
internal/module/cjs/loader.js:596
throw err;
Err: cannot find the module'express'
at.Function.Module._resolveFilename(internal/module/cjs/loader.js:594:15)
at.Function.Module._load(internal/module/cjs/loader.js:594:25)
and at the end it is showing
[nodemon] app crashed--waiting for file changes before starting...
even though i have install express earlier so we have to install there dependencies again
does this mean i haven't installed nodemon but after you told me i have installed it globally

**Comment by mysticatea** [comment]
You are encounting the usage problem of `npm` (Node.js package manager).

You have the following `dependencies` in package.json:

```
"dependencies": {
"gravatar": "^1.6.0"
},
"devDependencies": {
"eslint": "^4.19.1"
}
```

This means you have `gravatar` and `eslint` in your project. You don't have `nodemon` and `espress`.

I'm closing this issue as not related to ESLint. This is a place to report ESLint bugs or request features.
Please use general forum site like stackoverflow.com to ask the question.

Thanks.
